### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Colin Jaques

### DISCUSSION THREAD

**Discussion by CoJaques** [heig-vd-dai-course/heig-vd-dai-course#129] [Show and tell]
https://github.com/CoJaques/Docker-training

![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/91125307/06c41083-4c5c-4a2d-850e-5be11006a797)

**Comment by ludelafo** [comment]
Everything looks good to me!
